avm/src/vm/opcode.rs

```diff
@@ -1560,90 +1556,80 @@ pub static OPCODES: Lazy<HashMap<i64, ByteOpcode>> = Lazy::new(|| {
   });
   cpu!("poparr", |args, hand_mem| {
     let last = hand_mem.pop(args[0]);
-    hand_mem.write_fractal(args[2], &Vec::new());
+    hand_mem.init_fractal(args[2]);
     if last.is_ok() {
       hand_mem.push_fixed(args[2], 1i64);
       let record = last.ok().unwrap();
-      let (a, b) = record;
-      hand_mem.push_idxs(args[2], a, b as usize);
+      hand_mem.push_register_from_fractal(args[2], &record, 0);
```

Comment:
This is such a weird method name. I wonder if there's any better name, or if it would be better to figure out a way to make smaller, more composable functions here?

Reply:
I don't think we should make a composable function out of this. That pattern is what lends itself to have the memory abstraction to be leaked

The name is long and specific, but it is much easier to follow and imo not weird. Personally, when I saw `push_idxs` after not looking at the codebase for a while, it was not obvious what was going on. I am okay to shorten/change the name, but would prefer to keep it as-is.

Reply:
hmmm `push_register_out` might be better